examples/app-dir-i18n-routing/app/[lang]/page.tsx

```diff
@@ -3,19 +3,15 @@ import { Locale } from '../../i18n-config'
 import Counter from './components/counter'
 import LocaleSwitcher from './components/locale-switcher'
 
-export default async function IndexPage({
-  params: { lang },
-}: {
-  params: { lang: Locale }
-}) {
+export default async function IndexPage({ params: { lang } }: { params: { lang: Locale } }) {
   const dictionary = await getDictionary(lang)
 
   return (
     <div>
       <LocaleSwitcher />
       <p>Current locale: {lang}</p>
       <p>
-        This text is rendered on the server:{' '}
```

Comment:
Why was the space removed?

Reply:
ah yes i'm sorry i just used VS code for the first time to contribute and ...))